spec/feature/feature_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "./lib/takeaway"
+
+describe 'User Stories' do
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+let(:menu2) {menu2 = {
+  "fish and chips" => 3.20, 
+  "burger and chips" => 4.20, 
+  "chips" => 1,
+  }
+}
+
+let(:menu3) {menu3 = {
+  "fish and chips x1" => 3.20, 
+  "burger and chips x2" => 8.40, 
+  "chips x3" => 3,
+  }
+}
+
```

Comment:
These `let` blocks work but contain something that is not required.  Can you figure out what that is, by playing with the code?

Reply:
Have now refactored the code, I did not need an additional variable assignment in the let blocks and my rspec tests still pass.